src/scss/includes/openingHour.scss

```
@@ -0,0 +1,13 @@
+.openingHour {
+  margin: 0;
+  font-size: 14px;
+  color: #353c52;
```

Not specific to this PR (other hard-coded colors remain here and there), but note that an existing variable could be used here:
```suggestion
  color: $primary_text;
```